backend/webserver/tests.py

```diff
@@ -620,3 +623,621 @@ def test_delete_404(self):
+            description="Testing post 3",
+            unlisted=False
+        )
+
+        Post.objects.create(
+            author=author,
+            title="Test Post 4",
+            description="Testing post 4",
+            unlisted=False
+        )
+
+        url = f'/api/authors/{author.id}/posts/'
+        self.client.force_authenticate(user=mock.Mock())
+        response = self.client.get(url, format="json")
+        self.assertEqual(response.status_code, status.HTTP_200_OK)
+        # since the posts are ordered in descending order the post that was last created should be first in the list
+        post_4 = response.data[0]
+        post_3 = response.data[1]
+        post_2 = response.data[2]
+        post_1 = response.data[3]
+        self.assertEqual(post_4["description"], "Testing post 4")
+        self.assertEqual(post_3["description"], "Testing post 3")
+        self.assertEqual(post_2["description"], "Testing post 2")
+        self.assertEqual(post_1["description"], "Testing post 1")
+ 
+    def test_send_friend_post_to_inbox(self):
+        author_1 = Author.objects.create(username="author_1", display_name="author_1")
+        author_1.set_password("pass123")
+        author_1.save()
+        request_payload = {"username": "author_1", "password": "pass123"}
+        self.client.post("/login/", data=request_payload, format="json")
+        author_2 = Author.objects.create(username="author_2", display_name="author_2")
+        author_3 = Author.objects.create(username="author_3", display_name="author_3")
+        author_4 = Author.objects.create(username="author_4", display_name="author_4")
+        author_5 = Author.objects.create(username="author_5", display_name="author_5")
+        Follow.objects.create(follower=author_2,followee=author_1)
+        Follow.objects.create(follower=author_3,followee=author_1)
+        Follow.objects.create(follower=author_4,followee=author_1)
+        Follow.objects.create(follower=author_5,followee=author_1)
+        payload = {
+            "title": "Mark McGoey",
+            "description": "new description",
+            "unlisted":True,
+            "content":"Some new content",
+            "visibility":"FRIENDS",
+            "content_type":"text/plain"
+        }
+        url = f'/api/authors/{author_1.id}/posts/'
+        self.client.force_authenticate(user=author_1)
+        response = self.client.post(url,data=payload)
+        self.assertEqual(response.status_code, status.HTTP_201_CREATED)
+        inbox = Inbox.objects.first()
+        self.assertEqual(author_2, inbox.target_author)
+        self.assertEqual(4, Inbox.objects.count())
+        self.assertEqual(payload["title"], inbox.post.title)
+        self.assertEqual(payload["description"], inbox.post.description)
+        self.assertEqual(payload["visibility"], inbox.post.visibility)
+        self.assertEqual(payload["content_type"], inbox.post.content_type)
+
+    def test_send_private_post_to_inbox(self):
```

Comment:
For this use case, another good question for the forum would be - to send a a private post, does the receiver have to follow the author creating the post? 
You're assuming that the answer is no over here, which is fine but I think it's worth clarifying.